src/test/java/io/confluent/connect/elasticsearch/integration/ElasticsearchConnectorBaseIT.java

```diff
@@ -28,22 +45,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
-import com.google.gson.JsonArray;
-import com.google.gson.JsonObject;
-import io.confluent.connect.elasticsearch.ElasticsearchClient;
-import io.confluent.connect.elasticsearch.ElasticsearchSinkConnector;
-import io.confluent.connect.elasticsearch.jest.JestElasticsearchClient;
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
-import org.apache.kafka.connect.json.JsonConverter;
-import org.apache.kafka.connect.storage.StringConverter;
-import org.apache.kafka.test.TestUtils;
-import org.junit.After;
-import org.junit.AfterClass;
-import org.junit.Before;
-
-public class ElasticsearchConnectorBaseIT extends BaseConnectorIT {
+public abstract class ElasticsearchConnectorBaseIT extends BaseConnectorIT {
```

Comment:
```suggestion
public class ElasticsearchConnectorBaseIT extends BaseConnectorIT {
```
unintended change.

Reply:
@sudeshwasnik Are we committing this change?